subsys/bluetooth/host/att.c

```diff
@@ -742,6 +742,7 @@ static struct net_buf *bt_att_chan_create_pdu(struct bt_att_chan *chan, uint8_t
 			timeout = K_FOREVER;
 		}
 	}
+	}
```

Comment:
Two `}` on consecutive lines with the same indentation level looks quite fishy, but I guess that's our coding style...

Reply:
Yeah, due to the lack of indent for cases :) (that always looked odd to me)